packages/legend-studio-manual-tests/src/__tests__/roundtrip-grammar/RoundtripGrammar.test.ts

```diff
@@ -52,7 +52,8 @@ const ENGINE_SERVER_URL = 'http://localhost:6060/api';
 const TEST_CASE_DIR = path.resolve(__dirname, 'cases');
 const EXCLUDED_CASE_FILES: string[] = [
   'basic-M2M.pure', // until we fix the grammar parser to not include the variable for the property mapping lambda
-  'embedded-relational-mapping-with-imports.pure', // TODO?
+  // 'embedded-relational-mapping.pure',
+  'nested-embedded-relational-mapping.pure',
```

Comment:
so these are now passing. nested-embedded and basic M2M?

Reply:
good point, it's working now, I will remove this exclusion in next PR